.changeset/fresh-radios-compete.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-kubernetes': patch
+'@backstage/plugin-kubernetes-backend': patch
+'@backstage/plugin-kubernetes-common': patch
```

Comment:
Actually breaking changes in both the backend and common right?

Reply:
fixed